src/omero_zarr/cli.py

```diff
@@ -206,38 +207,19 @@ def _do_export(self, path, name, args):
         if args.max_workers:
             options += " --max_workers=" + args.max_workers
 
-        print(options)
+        self.ctx.dbg("%s$ bin/bioformats2raw %s %s %s" % (bf2raw,
+                                                            options,
+                                                            abs_path.resolve(),
+                                                            target.resolve()))
         process = subprocess.Popen(
-            ["bin/bioformats2raw", options, abs_path, target],
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-            cwd=bf2raw,
-        )
+            ["bin/bioformats2raw", options, abs_path.resolve(),
```

Comment:
Another option might be to drop `bin/` here, make `--bf` a true/false flag, and leave the user to set `$PATH` correctly.

Reply:
Last time I was testing it, it only worked if the working directory really was the main bioformats2raw directory and it  was exectuted as `bin/bioformats2raw ...` because otherwise it couldn't find the jar files. But I'll try again. Simply calling `bioformats2raw` from the `$PATH` would definitely be the nicer variant.